pkg/csi/cinder/controllerserver.go

```diff
@@ -103,6 +103,12 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 
 	// Volume Create
 	properties := map[string]string{"cinder.csi.openstack.org/cluster": cs.Driver.cluster}
+	//Tag volume with metadata if present: https://github.com/kubernetes-csi/external-provisioner/pull/399
```

Comment:
looks like `--extraCreateMetadata` is needed for such behavior to take effect ,so do we want to enable that in the mnaifest?

https://github.com/kubernetes-csi/external-provisioner/pull/399/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R68

Reply:
the flag is called `--extra-create-metadata` by now. I added it to the chart and the manifests.

Reply:
awesome, thanks!